pynws/nws.py

```diff
@@ -87,48 +111,60 @@ async def get_detailed_forecast(self: Nws) -> DetailedForecast:
         """
         if self.wfo is None:
             await self.get_points()
+        if self.wfo is None or self.x is None or self.y is None:
```

Comment:
Either this should move into get_points, or it should be inside the prior if statement, which is expanded to check for `self.x` and `self.y`, or both.

Reply:
Without those checks, mypy has the following complaints. I could use `cast` to remove the `Optional` in the arguments to `raw_detailed_forecast` instead if you prefer.
```
pynws\nws.py:117: error: Argument 1 to "raw_detailed_forecast" has incompatible type "Optional[str]"; expected "str"
pynws\nws.py:117: error: Argument 2 to "raw_detailed_forecast" has incompatible type "Optional[int]"; expected "int"
pynws\nws.py:117: error: Argument 3 to "raw_detailed_forecast" has incompatible type "Optional[int]"; expected "int"
```